perception_dataset/rosbag2/rosbag2_to_annotated_t4_tlr_converter.py

```diff
@@ -250,12 +250,12 @@ def _convert_traffic_lights(self, scene_timestamp_objects_pair_list: List[Dict[s
         )
 
     def _process_traffic_lights(
-        self, message: Union[TrafficLightRoiArray, TrafficSignalArray]
+        self, message: Union[TrafficLightRoiArray, TrafficLightArray]
     ) -> List[Dict[str, Any]]:
         """
         Args:
           message: autoware_auto_perception_msgs.msg.TrafficLightRoiArray
-            or autoware_auto_perception_msgs.msg.TrafficSignalArray
+            or autoware_auto_perception_msgs.msg.TrafficLightArray
```

Comment:
@miursh @ktro2828 (Not directly related to this PR, but) do you know if these lines are correct or not? 

```
        Args:
          message: tier4_perception_msgs.msg.TrafficLightRoiArray
            or tier4_perception_msgs.msg.TrafficLightArray
```

Reply:
`autoware_auto_perception_msgs.msg.TrafficSignalArray`  is going to be renamed as `autoware_auto_perception_msgs.msg.TrafficLightGroupArray`. 
Thus, please maintain autoware_auto_perception_msgs.msg.TrafficSignalArray as it is for now.

Reply:
(Not change in this PR)
I'm wondering if autoware_auto_perception_msgs.msg.TrafficSignalArray  is the correct argument...

Reply:
(Sorry, maybe my answer above also made a confusion)
The argument of this function is from `tier4_perception_msgs` and not `autoware_auto_perception_msgs` at least from the implementation, which was also intended in the above comment: https://github.com/tier4/tier4_perception_dataset/pull/80#discussion_r1493907300

Reply:
I see. In that case, the comment should be tier4_perception_msgs. 
However, I found that `Rosbag2ToAnnotatedT4TlrConverter` does not work well, so I will fix this in another PR.